lib/data/slackarchive.py

```diff
@@ -1,61 +1,157 @@
+        session = requests.Session()
+        messages = session.get(url, headers=self.origin_header)
+        return json.loads(messages.content.decode('utf-8'))['messages']
+
+    def fetch_users(self, team_id, channel_id, offset):
+        """
+        This method returns a list of users from a channel according to the range specified by the size
+        and offset parameters.
+        :param self
+        :param team_id: The unique team identifier as returned by the fetch_team method.
+        :param channel_id: The unique channel identifier as returned by the fetch_channel method.
+        #:param size: The number of messages to fetch, in a reverse-chronological order.
+        :param offset: The number of messages to skip while fetching, in a reverse-chronological order. An
+        offset of 5 with a size of 100 will fetch the 100 messages after the 5 latest messages in a channel.
+        :return: A dict containing a list of messages from the specified channel, in a reverse-chronological
+        order, according to the range specified by the size and offset parameters.
+        """
+        size = 100
+        url = "https://api.slackarchive.io/v1/messages?size=%d&team=%s&channel=%s&offset=%d" % (size, team_id, channel_id, offset)
+        session = requests.Session()
+        messages = session.get(url, headers=self.origin_header)
+        return json.loads(messages.content.decode('utf-8'))['related']['users']
+
+
+class DataToJson(object):
 
+    def __init__(self, team_name):
+        """
+        This method initialises the team_name variable and instantiates an object of 
+        DataLogsLoader() class.
+        :param self
+        :param team_name: It is the team name of which the data logs have to downloaded.
+        """
+        self.team_name = team_name
+        self.data_loader = DataLogsLoader(self.team_name)
 
-def fetch_messages(domain, team_id, channel_id, size, offset):
+    def team_to_json(self):
+        """
+        This method writes the team data logs into a file data/team-name/team.json
+        :param self
+        """
+        self.team_data = self.data_loader.fetch_team()
+        write_to_json('../../data/%s/team.json' % self.team_name, self.team_data)
+
+    def channels_to_json(self):
+        """
+        This method writes the channels data log of the given team to data/team-name/channels.json
+        :param self
+        """
+        self.channels_data = self.data_loader.fetch_channels(self.team_data['team_id'])
+        write_to_json('../../data/%s/channels.json' % self.team_name, self.channels_data)
+    
+    def messages_to_json(self):
+        """
+        This method writes the messages data logs of each channel of the team in json files. 
+        These files are kept in data/team-name/messages/channel-name.json
+        :param self
+        """
+        list_messages = []
+        for channel in self.channels_data:
+            for offset in range(0, 1000, 100):

```

Comment:
try not to hardcode values. Use config file to store these.

Reply:
I will make a config file to replace all the hard code above